server/src/api/controllers/app.controller.ts

```diff
@@ -0,0 +1,8 @@
+import { Request, Response } from 'express';
+import { success } from '../../../node-mongo-helpers';
+
+export const getAppController = async (req: Request, res: Response) => {
+  const message = 'App works!';
+  success(message);
```

Comment:
You did not make the change requested here @samuko-things to change this message to "Collabo Web app".